src/components/navigation/GlobalNavigation/global-navigation.css

```diff
@@ -295,6 +290,19 @@
   cursor: pointer;
 }
 
+.globalNavigation__suiteLogo-background {
+  background-color: var(--mp-brand-primary-2);
```

Comment:
nit: hmmm, I'd prefer to see semantic names, but it's up to you (I'm not the CSS owner here 😝 )
```suggestion
  background-color: var(--color-primary-bg-hover);
```

Reply:
<img width="295" alt="Screenshot 2024-05-21 at 11 45 20 PM" src="https://github.com/mParticle/aquarium/assets/17934324/d7bd2379-e6d0-4b9a-8d6b-8f250f680a72">

jeje I used this variable to keep consistency w the icons :)